Add support for alarm platform

KJonline opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->

Integrate hive home shield alarm with the home assistant `alarm_control_panel`.

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/48703
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Commit:
Add support for alarm platform

homeassistant commented:
Hi @rickylee64, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17294).

Once you do that we will be able to review and accept this pull request.

Thanks!

frenck reviewed (dismissed):
✅ Approved. Can be merged as soon as the parent PR gets merged.

TomBrien committed:
Use consistent thousands separator in UX blog post (#19767)

ludeeus committed:
Change name from Uptime Robot to UptimeRobot (#19775)

ronalterde committed:
Add onewire support for DS2413 (#19337)

balloob committed:
2021.10.5

MartinHjelmare committed:
Add vlc_telnet config flow (#19733)

MartinHjelmare committed:
Update zwave_js migration docs (#19432)

cturra committed:
adding default to round() examples to be compliant new validation (#19800)

cturra committed:
updating weather template example (#19799)

calebmah committed:
Update instructions for Automation Editor (#19770)

mib1185 committed:
Refer to the official VLC add-on in VLC telnet integration (#19808)

walzert committed:
Update doc after getting error in file editor (#19815)

apworks1 committed:
Update tuya.markdown (#19794)

danielrheinbay committed:
Add TLS and utf8mb4 for MariaDB (#19793)

gjong committed:
Update youless supported hardware devices (#19789)

emontnemery committed:
Document entity_category for MQTT entities (#19786)

ColinRobbins committed:
Fix text for boost off (#19776)

johan12345 committed:
Update example configuration for HP iLO server health sensor (#19820)

bdraco committed:
Add support for power state push updates to flux_led (#19813)

balloob committed:
2021.10.6 (#19829)

dependabot[bot] committed:
Bump tzinfo-data from 1.2021.3 to 1.2021.4 (#19830)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump actions/checkout from 2.3.4 to 2.3.5 (#19831)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

atxbyea committed:
Update yeelight.markdown (#19837)

atxbyea committed:
Update tplink.markdown (#19836)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

depuits committed:
Fixed contradiction for multiple entity IDs (#19840)

atxbyea committed:
Add interference solution for Z-Wave JS troubleshooting (#19841)

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

Hedda committed:
Updated zigate URLs + removed zigate-ttl + added new zigate-ethernet and ZiGate USB to zha.markdown (#19846)

chemelli74 committed:
Fritz: small clarification + fixed leftover (#19844)

frenck committed:
Merge remote-tracking branch 'origin' into next

ludeeus committed:
Rename unsupported reason container to software (#19857)

ludeeus committed:
Use configuration_basic for Glossary (#19839)

ludeeus committed:
Add OS-agent to diagnostics (#19859)

MartinHjelmare committed:
Update Z-Wave integrations deprecation and migration status (#19856)

ikifar2012 committed:
Added links to Compare Installation Methods (#19827)

rytilahti committed:
improve kasa provisioning doc (#19855)

OGKevin committed:
Reduce table size for xiaomi_miio vacuum sensor. (#19709)

* Reduce table size for xiaomi_miio vacuum sensor.

Signed-off-by: Kevin Hellemun <17928966+OGKevin@users.noreply.github.com>

* Fix spelling.

Signed-off-by: Kevin Hellemun <17928966+OGKevin@users.noreply.github.com>

dependabot[bot] committed:
Bump racc from 1.5.2 to 1.6.0 (#19869)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Kane610 committed:
UniFi - Add service reconnect_client (#19738)

aladin2000 committed:
- TX LF37E30 ( can't power on) (#19890)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Megachip committed:
Update bh1750.markdown (#19894)

pattyland committed:
Updated Link to Raspberry Pi documentation (#19899)

stefanroelofs committed:
Fix markdownlint / spelling  issues #1 (#19900)

shbatm committed:
Add push_weather_data Service to Rainmachine (#19600)

Co-authored-by: Aaron Bach <bachya1208@gmail.com>
Co-authored-by: Franck Nijhof <git@frenck.dev>

allenporter committed:
Add documentation for Nest Battery Cameras and Doorbells and WebRTC streams (#19664)

Co-authored-by: Franck Nijhof <git@frenck.dev>

ANMalko committed:
Add LOOKin integration (#19884)

Co-authored-by: J. Nick Koston <nick@koston.org>

rikroe committed:
Add BMW deactivate_air_conditioning service (#19864)

Commit:
Support connecting to ElkM1 over TLS 1.2 (#19547)

frenck committed:
Add service configuration URL to MQTT (#19777)

* Add service configuration URL to MQTT

* Update light.mqtt.markdown

Co-authored-by: Erik Montnemery <erik@montnemery.com>

starkillerOG committed:
Add Bypassing UDP multicast to Motionblinds (#19850)

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

jbouwh committed:
MQTT Alarm Control Panel - Remote code validation (#19788)

Co-authored-by: Erik Montnemery <erik@montnemery.com>

bieniu committed:
Add information about purifiers `Use Time` sensor, Xiaomi Miio (#19803)

bachya committed:
Add docs for SimpliSafe OAuth flow (#19637)

frenck committed:
Add tamper device class for binary sensor (#19755)

OGKevin committed:
Use configuration_basic to document xiaomi vacuum sensors. (#19873)

stefanroelofs committed:
Spell check dict update (#19730)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

stefanroelofs committed:
Fix markdownlint / spelling issues #2 (#19903)

bachya committed:
Add docs re: disabled-by-default RainMachine entities (#19759)

Hedda committed:
Update zha.markdown to mention and explain binding and unbinding (#16336)

Co-authored-by: Franck Nijhof <git@frenck.dev>
Co-authored-by: Alexei Chetroi <lexoid@gmail.com>

flabbamann committed:
Add documentation for fritz lights (#18451)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

nagyrobi committed:
Add some clarification to Fronius integration docs (#18888)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>
Co-authored-by: Matthias Alphart <farmio@alphart.net>

Hedda committed:
Update zha.markdown with note about changing Zigbee channel (#18865)

Co-authored-by: Franck Nijhof <git@frenck.dev>
Co-authored-by: Alexei Chetroi <lexoid@gmail.com>

Petro31 committed:
Add template today_at function/filter docs. (#19804)

raman325 committed:
Document area_entities/area_devices template functions/filters (#19080)

pattyland committed:
Nello Removal (#19823)

koskila committed:
Improve documentation for history-graph #19893 (#19895)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

SNoof85 committed:
Solaredge, update template sensors examples (#19909)

elupus committed:
Add mention of additonal devices for 0.12 (#18852)

koying committed:
add state_class to rest sensor & device_class to rest switch (#19860)

erffrfez committed:
add reference Riemann integral in "Understanding Home Energy Management" (#19251)

jjlawren committed:
Improve Plex library sensor description and add automation example (#19740)

drinfernoo committed:
Update Watson TTS Documentation (#19912)

Snailboy2 committed:
Updated links (#19915)

danielrheinbay committed:
Fix IP address typo in cast.markdown (#19916)

M-J-W-H committed:
changes BLTE to BTLE (#19927)

delorean-dev committed:
Update epsonworkforce.markdown (#19926)

michaeldavie committed:
Update environment_canada for config flow (#19564)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

koying committed:
Service call with a native dictionary (#19922)

jjlawren committed:
Describe Sonos feature switches (#18897)

Co-authored-by: Franck Nijhof <git@frenck.dev>

erlendmoland committed:
Removed hassio link reference, clarified text (#19938)

erlendmoland committed:
Updated link from /hassio/ to /getting-started/ (#19937)

erlendmoland committed:
Updated outdated links on troubleshooting page (#19939)

bdraco committed:
Update supported devices for gogogate2 (#19935)

jjlawren committed:
Remove default disabled note for Sonos switches (#19931)

shawnshyguy committed:
add "car" to Traccar categories (#19930)

ketilmo committed:
Corrected dead link (#19929)

ketilmo committed:
Corrected typo in link to integrations (#19928)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

farmio committed:
Configurable KNX Number mode (#19943)

EgonMarmol committed:
Updated outdated URLs on Authentication Providers page (#19942)

ketilmo committed:
Gas: Updated SlimmeLezer and link to SlimmeLezer+ (#19944)

erlendmoland committed:
Updated Arilux repo reference (#19945)

ketilmo committed:
Electricity grid: Updated SlimmeLezer and link to SlimmeLezer+ (#19951)

Commit:
Add entity category to knx (#19953)

abmantis committed:
Update Whirlpool supported ACs list (#19954)

frenck committed:
Add support for unit of measurement in MQTT number platform (#19948)

EgonMarmol committed:
Refereshed links (#19950)

stefanroelofs committed:
Fix broken link (#19958)

QbaF committed:
Update netatmo.markdown (#19959)

ludeeus committed:
Add running device class to binary sensor (#19963)

raman325 committed:
Update zwave_js.ping documentation (#19972)

frenck committed:
Fix forum top Blueprint URLs (#19979)

lewei50 committed:
change units (#19975)

ludeeus committed:
Rename the google calendar docs to just google (#19980)

bdraco committed:
Add fan groups (#19853)

plasota committed:
Add info about `index` config to notify documentation (#19722)

Co-authored-by: Przemyslaw (Pem) Lasota <plasota@users.noreply.github.com>

dependabot[bot] committed:
Bump tzinfo-data from 1.2021.4 to 1.2021.5 (#19955)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

mezz64 committed:
Update Hikvision docs (#19978)

frenck committed:
Merge branch 'current' into next

bachya committed:
Restore SimpliSafe event documentation (#19867)

ggravlingen committed:
Include support for Tradfri STARKVIND Air purifier (#19925)

garbled1 committed:
Update venstar.markdown for config_flow (#19888)

jjlawren committed:
Update Plex to reflect advanced searches (#19401)

Co-authored-by: Franck Nijhof <git@frenck.dev>

Danielhiversen committed:
Update opengarage (#19877)

Co-authored-by: Franck Nijhof <git@frenck.dev>

rfleming71 committed:
Octoprint Config Flow and sensors (#19854)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

spacegaier committed:
Update dev-tools states: Tab order and wildcard search (#19361)

Co-authored-by: Franck Nijhof <git@frenck.dev>

mishmanners committed:
Update config for scenes (#19605)

Co-authored-by: Franck Nijhof <git@frenck.dev>

jugla committed:
Add SQL example to get former value (#19105)

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

MartinHjelmare committed:
Rework Z-Wave docs (#19962)

Co-authored-by: Franck Nijhof <git@frenck.dev>

RobertMe committed:
Add documentation for supplying offset to time trigger (#19536)

avee87 committed:
Add documentation for Supervisor integration (#19778)

davidjb committed:
Only run periodic GitHub Actions in main repository (#19779)

peyanski committed:
Update conditions.markdown (#18725)

flabbamann committed:
Add fritz lights to tested devices (#19987)

merc1031 committed:
Add note extended cron syntax note croniter (#19541)

Mariusthvdb committed:
add reload events for Mqtt and Template (#19505)

cturra committed:
updating utility_meter example to use new template format (#19826)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

SNoof85 committed:
serial update template sensor example (#19969)

SNoof85 committed:
garadget update template sensor example (#19971)

SNoof85 committed:
withings update template sensor example (#19968)

SNoof85 committed:
solax update template sensor example (#19967)

SNoof85 committed:
pvoutput update template sensor example (#19966)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

jim-granger committed:
Update dsmr.markdown (#19997)

SNoof85 committed:
update uk_transport template sensor examples (#19965)

Petro31 committed:
Add template average filter and function. (#19805)

CCOSTAN committed:
Hive is shutting down North American Servers. (#19999)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

JamesW-AU committed:
Update pvoutput.markdown (#20001)

davet2001 committed:
Update aurora_abb_powerone documentation for config flow setup (#13748)

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

dependabot[bot] committed:
Bump remark-frontmatter from 4.0.0 to 4.0.1 (#20004)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

oischinger committed:
vicare: Remove circuit config option (#19814)

Bre77 committed:
Add select platform (#18963)

Commit:
Microsoft tts defaults (#19998)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

zlinoliver committed:
update tuya markdown with content and image (#19989)

stefanroelofs committed:
Fix markdownlint / spelling  issues #3 (#19961)

frenck committed:
Merge branch 'current' into next

frenck committed:
Remove Huawei Router (ADR-0004) (#19991)

frenck committed:
Reflect support platforms for Tuya (#20006)

dgomes committed:
Warns Utility Meters users that offset < 28 days (#20009)

axilleas committed:
Refactor the SPI and ums info for the ODROID-N2+ (#20007)

ludeeus committed:
Add missing hassio entities (#20005)

SNoof85 committed:
edimax update template sensor example (#19970)

emontnemery committed:
Correct documentation for template entity availability template (#20011)

oischinger committed:
Improve Vicare setmode service parameter documentation (#19993)

chemelli74 committed:
Add new Fritz service (#20013)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

drizzle1 committed:
Add pi4 and pi4B boards to orangepi_gpio (#19982)

Co-authored-by: Pascal Roeleven <pascal.roeleven@hotmail.com>
Co-authored-by: J. Nick Koston <nick@koston.org>

frenck committed:
Merge branch 'current' into next

hesselonline committed:
Adding a number component to the wallbox integration. (#18446)

frenck committed:
2021.11: Initial beta release notes

frenck committed:
Merge branch 'next' into rc

frenck committed:
Sync codebase with docs for 2021.11

pixeldoc2000 committed:
Update telegram_bot.markdown (#20022)

SNoof85 committed:
vesync update template sensors examples (#20018)

SNoof85 committed:
waze_travel_time update template sensor example (#20019)

SNoof85 committed:
time_date update template sensor example (#20021)

Bre77 committed:
Repo for croniter has moved (#20025)

SNoof85 committed:
transport_nsw update template sensor example (#20020)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

zlinoliver committed:
update Tuya doc (#20026)

stefanroelofs committed:
Fix markdownlint / spelling  issues #4 (#20033)

Chen-IL committed:
Use modern template binary-sensor in example (#20029)

briglx committed:
Format powerwall sensors. Match to code. (#20035)

javicalle committed:
Add Bitron Video/Smabit BV AV2010/10 link (#20038)

rytilahti committed:
Document wled number platform (#20039)

rytilahti committed:
Document wled number platform (#20039)

SeanPM5 committed:
Remove outdated Z-Wave paragraph from Config docs (#20043)

SeanPM5 committed:
Instructions on how to add Counters via UI (#20044)

SeanPM5 committed:
Fix text pointing to wrong "Edit" location of docs (#20042)

natekspencer committed:
Updated Litter-Robot documentation for new select entity (#19936)

allenporter committed:
Change order of nest instructions in fewer steps (#20073)

SNoof85 committed:
Update ecovacs template sensor & binary_sensor examples (#20050)

axilleas committed:
Update to Node.js 16, the latest LTS (#20064)

SNoof85 committed:
Update imap_email_content template sensor examples (#20054)

SNoof85 committed:
Update homematic templates examples (#20053)

emontnemery committed:
Correct template select docs (#20055)

peternijssen committed:
Fix configuration block on the Scenes page (#20056)

lindsaymarkward committed:
Improve wording for android_ip_webcam (#20095)

drthanwho committed:
Clarify check_config script is only for Core installations (#20100)

chrisns committed:
Update nest.markdown to define the expiry period (#20067)

* Update nest.markdown to define the expiry period

the expiry period for a subscription by default is 31 days, which means the integration breaks after 31 days :(
updating the docs to make it clear this should be set to not expire

* Update nest.markdown

* Update nest.markdown

* Update source/_integrations/nest.markdown

Co-authored-by: Allen Porter <allen.porter@gmail.com>

Co-authored-by: Allen Porter <allen.porter@gmail.com>

lindsaymarkward committed:
Improve wording for HomeKit (#20096)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

balloob committed:
2021.10.7

balloob committed:
Link PRs for 10.7

emontnemery committed:
Revert "Add documentation for supplying offset to time trigger" (#20119)

emontnemery committed:
Revert "Add documentation for supplying offset to time trigger" (#20119)

cogneato committed:
Update usb_boot.markdown (#20120)

jbouwh committed:
Add command_template to MQTT select platform (#20117)

epenet committed:
Drop requirement for Renault API keys (#20122)

jbouwh committed:
Add command_template to MQTT number platform (#20121)

koying committed:
Add REST sensor/binary_sensor/switch templated headers & params (#20058)

frenck committed:
Merge branch 'current' into next

frenck committed:
Merge branch 'current' into rc

cogneato committed:
Update usb_boot.markdown again (#20123)

dependabot[bot] committed:
Bump actions/checkout from 2.3.5 to 2.4.0 (#20131)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump i18n from 1.8.10 to 1.8.11 (#20130)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

ikifar2012 committed:
Additional info for Flashing an Odroid-N2+ (#19828)

emontnemery committed:
Remove documentation for deprecated MQTT sensor last_reset (#20133)

frenck committed:
Merge branch 'current' into next

frenck committed:
Merge branch 'current' into rc

frenck committed:
2021.11: Update & sort breaking changes section

frenck committed:
2021.11: Update changelog

frenck committed:
2021.11: Add daylight saving time bug section

frenck committed:
2021.11: Visit device & service section

frenck committed:
2021.11: Tuya update

frenck committed:
2021.11: Tweaks

frenck committed:
2021.11: Intro & TOC

frenck committed:
2021.11: Icon picker section

frenck committed:
2021.11: Entity category section

frenck committed:
2021.11: Tweaks

frenck committed:
2021.11: Template section

frenck committed:
2021.11: Other noteworthy changes

frenck committed:
2021.11: Tweaks

frenck committed:
2021.11: Review comments

frenck committed:
2021.11: Review comments

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

frenck committed:
2021.11: Latest changelog

frenck committed:
Merge pull request #20137 from home-assistant/rc

frenck committed:
2021.11: Fix missing Vicare breaking change

OttoWinter committed:
Fix typo in 2021.11 release notes (#20140)

Chupaka committed:
Fix a typo (#20139)

emontnemery committed:
Update recorder documentation according to ADR18 (#20145)

Chupaka committed:
its != it's 😊 (#20141)

emontnemery committed:
Adjust minimum supported version of SQLite (#20152)

frenck committed:
Add button entity (#20105)

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

allenporter committed:
Rewrite nest OAuth instructions using Desktop App Auth (#20143)

* Add instructions for nest Desktop auth

Add instructions for configuring integration Desktop Auth. Using desktop auth simplifies configuration for end users by not requiring complex url setup including SSL and publicly resolvable dns names, etc.

* Update URL authentication

* Remove unnecessary whitespace changes

* Fix typo in oauth device instructions

balloob committed:
2021.11.1

garbled1 committed:
Update venstar.markdown for binary_sensor addition (#20104)

MartinHjelmare committed:
Remove migration section from ozw (#20162)

nick-seward committed:
docs: Adding to the Slack integration examples (#20172)

* docs: Adding to the Slack integration examples

I'm proposing two new examples on how to message someone directly in Slack and how to mention (ex. @user) a user in a Slack channel.

* doc: Fixing spelling mistake

slovdahl committed:
Add link to docs for condition trigger ID (#20169)

farmio committed:
Add decoded telegram payload value to `knx_event` (#20190)

* knx_event type key

* caps KNX

* Update source/_integrations/knx.markdown

Co-authored-by: Marvin Wichmann <marvin@fam-wichmann.de>

koying committed:
Allow overriding ensure_ascii in the "to_json" template filter  (#20218)

* to_json: add ensure_ascii

* Update source/_docs/configuration/templating.markdown

Co-authored-by: Erik Montnemery <erik@montnemery.com>

* lint

Co-authored-by: Erik Montnemery <erik@montnemery.com>

Smeagolworms4 committed:
Mqtt add object_id (#20217)

* Add MQTT object_id option

* add object_id doc each entities type

* Spelling

* Update source/_docs/mqtt/discovery.markdown

Co-authored-by: Erik Montnemery <erik@montnemery.com>

* add object_id doc each entities type

* add object_id doc each entities type

Co-authored-by: Erik Montnemery <erik@montnemery.com>

balloob committed:
2021.11.1

ludeeus committed:
Add integration file with metadata for energy (#20232)

epenet committed:
Remove Dyson integration (#20231)

Co-authored-by: epenet <epenet@users.noreply.github.com>

lambtho12 committed:
IOTA removal (#20224)

zlinoliver committed:
update github links for Tuya (#20225)

fabricioavil committed:
Update entity used in the "STARTUP" example (#20223)

emontnemery committed:
Revert "Mqtt add object_id" (#20233)

frenck committed:
Merge branch 'current' into next

emontnemery committed:
Document Mqtt object_id (#20235)

emontnemery committed:
Document MQTT button (#20219)

farmio committed:
Add Button platform for KNX (#20237)

* knx button

* eger

frenck committed:
Update Twente Milieu documentation for the removed service (#20291)

Commit:
Merge branch 'feature/add-hive-alarm-support' of https://github.com/Pyhive/home-assistant.io into feature/add-hive-alarm-support

* 'feature/add-hive-alarm-support' of https://github.com/Pyhive/home-assistant.io:
  Add support for alarm platform
  Add support for alarm platform

homeassistant commented:
Hi @depuits, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17294).

Once you do that we will be able to review and accept this pull request.

Thanks!

KJonline commented:
Closing as rebase it contaminated
